routes/forums.js

```diff
@@ -77,6 +76,9 @@ router.get('/:forum_id', (req, res) => {
         .then((forumData) => {
             info.forum = forumData;
             console.log(info);
+            info.forum.contentHTML = forumData.content
+                .replace(/#([^\[]+)\[([^\]]+)\]/g, (match, name, url) => `<a target='_blank' alt='${name}' href='${url}'>${name}</a>`)
+                .replace(/@([\w-]+)/g, (match, username) => `<a target='_blank' alt='${username}' href='#'>${username}</a>`);
```

Comment:
@tsangjustin for now nowhere. But just a cool reach features to have. Nice to see the usernames highlighted